python-threatexchange/threatexchange/exchanges/tests/test_state_compatibility.py

```diff
@@ -147,8 +147,23 @@ def get_NCMECCheckpoint() -> t.Tuple[NCMECCheckpoint, t.Sequence[object]]:
     ## Current
     max_ts = 1197433091
 
+    current = NCMECCheckpoint(
+        get_entries_max_ts=max_ts, next_fetch="", last_fetch_time=0
+    )
+
     # 1.0.x
-    current = NCMECCheckpoint(get_entries_max_ts=max_ts)
+    @dataclass
+    class NCMECCheckpointWithoutNext(FetchCheckpointBase):
+        """
+        0.99.x => 1.0.0
```

Comment:
blocking: This version isn't quite correct, the [current version](https://fb.workplace.com/groups/366137801110062) is 1.2.3

Reply:
fixed!